package.json

```diff
@@ -157,10 +158,10 @@
     "@types/lodash.throttle": "^4.1.7",
     "@types/lodash.uniqby": "^4.7.7",
     "@types/moment": "^2.13.0",
-    "@types/react": "^18.0.8",
-    "@types/react-dom": "^18.0.3",
-    "@types/react-image-gallery": "^1.0.5",
-    "@types/react-is": "^17.0.3",
+    "@types/react": "^18.2.55",
```

Comment:
React Aria packages use `React.JSX` in their typings, instead of the deprecated global `JSX`. That's why we needed to bump `@types/react` to the latest version in v18 branch: our previous version `@types/react@^18.0.3` didn't have `React.JSX` (see https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64464).